openwpm/config.py

```diff
@@ -133,6 +138,12 @@ class ManagerParams(DataClassJsonMixin):
     """A watchdog that tries to ensure that no Firefox instance takes up too much memory.
     It is mostly useful for long running cloud crawls"""
     process_watchdog: bool = False
+
+    """A watchdog that serves as a happy medium between killing a browser after each 
+    crawl and allowing the application to still perform quickly. Used as a way to save space 
+    in a limited environment with minimal detriment to speed."""
+    storage_watchdog_enable: bool = False
```

Comment:
What if this was an `Optional[int]`? So if you set a max size in byte, then the watchdog gets activated?

Reply:
I simply modeled it after the other watchdogs. Made the change now and added some error checking in the object's "run" function.